src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -775,6 +773,41 @@ def update_os_patch_configuration_sub_setting(self, patch_configuration_sub_sett
             self.composite_logger.log_error(error_msg)
             self.status_handler.add_error_to_status(error_msg, Constants.PatchOperationErrorCodes.DEFAULT_ERROR)
             raise
+
+    def revert_auto_os_update_to_system_default(self):
+        """ Reverts the auto OS update patch state on the machine to it's system default value, if one exists in our backup file """
+        self.composite_logger.log("Reverting the current automatic OS patch state on the machine to it's system default value before patchmode was set to 'AutomaticByPlatform'")
+
+        if not os.path.exists(self.os_patch_configuration_settings_file_path):
+            self.composite_logger.log_debug("Automatic OS patch config file not found on the VM. We won't be able to revert auto OS patch settings to their system default values")
+            return
+
+        self.__get_current_auto_os_updates_setting_on_machine()
+        self.composite_logger.log_debug("Logging current configuration settings for auto OS updates. [{0}={1}][{2}={3}]"
```

Comment:
[APM] in front of logs from this file.
Similarly elsewhere.

If there's a consistent string being used across package managers, it's good to encode them into Constant enum backports and reference for cross-PM consistency always. Like may this this one: ""Since the backup is invalid or does not exist, we won't be able to revert auto OS patch settings to their system default value"  -- you'll still need to prefix, but the rest of it changes across PMs as one.

These constants could be in the PackageManager base class as an alternative to central constants. (no particular preference)